spec/models/profile_spec.rb

```diff
@@ -4,11 +4,76 @@
   let(:user) { create(:user, :signed_up) }
   let(:another_user) { create(:user, :signed_up) }
   let(:profile) { create(:profile, user: user) }
-
-  subject { profile }
+  let(:pii) do
```

Comment:
They are indeed used a lot, but that doesn't mean we have to keep doing that. I've stopped using them, and I don't like `before` blocks when there is only one `it` block.